Profile/Profile.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>S-Beauty</title>
+    <link rel="stylesheet" href="/index.css">
+    <link rel="stylesheet" href="/Signup/Signup.css">
+    <link rel="stylesheet" href="https://madeui.netlify.app/components/Avatars/Avatar.css">   
+    <script src="https://kit.fontawesome.com/fda6e23714.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/Profile/Profile.css">
+</head>
+<body>
+ <div class="signup__page">
+    <header class="header">
+        <div class="logo__main">GLAM</div>
+        <div class="header__items">
+           <li><i class="fas fa-bars " id="menu__bar"></i></li>
+            <ul class="header__list">
+            <li ><a href="/Products/Products.html">Products</a></li>
+            <li><a href="/Cart/Cart.html"><i class="fas fa-shopping-cart"><div class="items__no">5</div></i></a></li>
+            <li ><a href="/Wishlist/Wishlist.html"><i class="fas fa-heart "><div class="items__no">2</div></i> </a></li>
+            <li ><a href="/Profile/Profile.html"><i class="fas fa-user"></i></a></li>
+           
+            
+            </ul>
+        </div>
+        </header>
+        <div class="profile__box">
```

Comment:
It would be nice to use input elements here instead of div. The rest looks good!